certified-connectors/Lang-ai/apiDefinition.swagger.json

```diff
@@ -163,7 +163,7 @@
                         "items": {
                           "type": "string"
                         },
-                        "description": "The list of features. The sign “>” indicates a second-level feature."
+                        "description": "The list of features. The sign \u201c>\u201d indicates a second-level feature."
```

Comment:
@evicentem9 Unicodes are not allowed as they would not be supported in all the Power Automate pages, Can you please let us know the reason to use unicodes here?

Reply:
this was an error, this should be changed. The original description without unicodes is correct.